app/views/comments/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h5> Create a new Comment:</h5>
+<%= form_with model: @comment, url: user_post_comments_url , method: :post do |form| %>
+   <div class="form-group">
+    <%= form.label :text , "Text" %>
+    <%= form.text_area :text , required: true %>
+   </div> 
+    <%= form.submit "Post Comment" %>
+<% end %>
```

Comment:
- [ ] Good job adding a form for comment 👏 . But please add one more form in the **views/posts/new.html.erb** file with inputs to make users able to create posts as required for this project and make sure that the forms created should function as the given [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES) 👍